tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -858,7 +867,7 @@ describe(__filename, () => {
     const params = { userId: user.id };
     renderUserProfile({ params, store });
 
-    sinon.assert.notCalled(dispatchSpy);
+    sinon.assert.neverCalledWith(dispatchSpy, loadUserAccount({ user }));
```

Comment:
The test description talks about it not fetching user reviews, which I don't think is the same thing as ` loadUserAccount`, so I think you need a different assertion here.